tests/port_cvar_BASE.csv

```diff
@@ -1,68 +1,68 @@
-8,0.06258656734327052
-9,0.07173983086290577
-10,0.08641134889991155
-11,0.08553753268702265
-12,0.08703171669403376
-13,0.07768676209151877
-14,0.08987765651186495
-15,0.0673532716275818
-16,0.07342886182612543
-17,0.07219397598501356
-18,0.085237728997843
-19,0.09299538346454014
-20,0.07395232497060067
-21,0.06968689155941896
-22,0.06625330578920541
-23,0.07749985675579167
-24,0.07895904355099637
-25,0.07378707933030614
-26,0.07105026363583872
-27,0.06309512033643717
-28,0.07308568301725987
-29,0.08014031036632616
-30,0.07712880396718737
-31,0.06348605291524358
-32,0.06561176836641255
-33,0.07738889994270287
-34,0.0760711117388937
-35,0.08932496741625659
-36,0.10189728343248686
-37,0.1231760296614636
-38,0.11676515441447959
-39,0.1088599326706746
-40,0.14289611665231197
-41,0.11355375537239434
-42,0.11202252905942578
-43,0.09815339384733139
-44,0.10996748941362544
-45,0.10319771481247687
-46,0.10875778065397822
-47,0.11582135787470475
-48,0.10795563202775757
-49,0.13578237344276653
-50,0.1397940983577706
-51,0.1326456882369138
-52,0.11802537660651964
-53,0.12069867673157761
-54,0.11202048025004732
-55,0.11562349453250925
-56,0.11332738735883328
-57,0.11370753615818235
-58,0.14462746613587474
-59,0.1296446995251748
-60,0.13547129805587585
-61,0.12077108032301379
-62,0.1091807050790592
-63,0.106504765787277
-64,0.09824836577451315
-65,0.14745013771755172
-66,0.12602297121497466
+0,0.0881588856198164
+1,0.08024193434958042
+2,0.07603692619092683
+3,0.07626816728717874
+4,0.07720150047995064
+5,0.06838057984766352
+6,0.0722838825019993
+7,0.06350304477057966
+8,0.06258707840436119
+9,0.07173974717899415
+10,0.08641134874585746
+11,0.08553753275365245
+12,0.08703171975510104
+13,0.07768676185652036
+14,0.08987765147611727
+15,0.0673532721046333
+16,0.07342886227835924
+17,0.07219397627660823
+18,0.0852377287682151
+19,0.09299538320545955
+20,0.07395232465888009
+21,0.06968689086328037
+22,0.06625330554286195
+23,0.07749985858982067
+24,0.0789590669682689
+25,0.07378670291331219
+26,0.07105189572641486
+27,0.06309512365063141
+28,0.07308568500046957
+29,0.08014030982179836
+30,0.07712880646485856
+31,0.06348605287855434
+32,0.0656119301841167
+33,0.0773890658955755
+34,0.0760711123497269
+35,0.08932496519648651
+36,0.13857713423912502
+37,0.12939979305688867
+38,0.11676515523915901
+39,0.16419721550585092
+40,0.142896115284607
+41,0.11355375695791009
+42,0.11202252909934332
+43,0.09815339097161604
+44,0.14767835417830766
+45,0.15121002158208455
+46,0.12812612422661984
+47,0.11582135838317076
+48,0.15332557348188508
+49,0.13578237316388386
+50,0.13979409747976015
+51,0.13264568622329137
+52,0.11802537961903746
+53,0.12069867632570366
+54,0.12567727628660214
+55,0.1156234930113666
+56,0.1383032025937626
+57,0.11370753868284231
+58,0.14462746442078703
+59,0.129644701666493
+60,0.13547129664541874
+61,0.12077107925960534
+62,0.10918070661379939
+63,0.11437340880275443
+64,0.1449110251286159
+65,0.14745013789187908
+66,0.13735899661551734
```

Comment:
It's not the same always but I guess it's ok because scenarios are different, right?